public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -147,6 +148,7 @@ export class DashboardModel {
     this.refresh = data.refresh;
     this.snapshot = data.snapshot;
     this.schemaVersion = data.schemaVersion || 0;
+    this.fiscalYearStartMonth = data.fiscalYearStartMonth || 0;
```

Comment:
super nit: Maybe use nullish coalescing operator instead? 
```suggestion
    this.fiscalYearStartMonth = data.fiscalYearStartMonth ?? 0;
```

Reply:
Ah, nice, I was just doing the same as the line above, coding on autopilot.